src/components/registerpage/Register/index.jsx

```diff
@@ -0,0 +1,253 @@
+import { Alert, Button, Checkbox, Input } from "antd";
+import Modal from "antd/lib/modal/Modal";
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { validateEmail } from "../registerHelpers";
+
+const MAX_PASSWORD_LENGTH = 8;
+
+const Register = ({
+  // check container/LoginRegister/index.jsx to see functions
+  // function
+  onSubmit,
+  // function
```

Comment:
delete comments, they're not useful. handleX is already known to be fn, same with onSubmit.

Reply:
PTAL